django_project_base/notifications/rest/notification.py

```diff
@@ -387,6 +410,7 @@ def filter_queryset_field(self, queryset, field, value):
 
     def get_serializer_class(self):
```

Comment:
Pri tem serializerju bi bila rešitev tudi precej lepša, če remaining emails in remaining SMSs ne bi bila nested serializerja, ampak samo polji v tem serializerju. Saj sta itaq calculated. Če bi ju pa zraven tega še v isto vrstico dal, bi bilo pa že kičasto :)

Reply:
odprt issue https://taiga.velis.si/project/velis-django-project-admin/issue/723